firestore/firestore_snippets/save.go

```diff
@@ -22,10 +23,11 @@ func addDocAsMap(ctx context.Context, client *firestore.Client) error {
 		"country": "USA",
 	})
 	if err != nil {
-		return err
+		// Handle the error correctly here.
+		log.Printf("An error has occurred: %s", err)
```

Comment:
It's more common to return the error here. It's OK to log and return then return `nil` below. The semantics are the same, but it's easier to read.

Reply:
If the return is inside the region tags but the function declaration isn't, the return doesn't make sense  in the scope of the snippet.